crates/openvino/src/request.rs

```diff
@@ -20,12 +20,13 @@ unsafe impl Sync for InferRequest {}
 
 impl InferRequest {
     /// Create a new [`InferRequest`] from [`ov_infer_request_t`].
+    #[inline]
     pub(crate) fn from_ptr(ptr: *mut ov_infer_request_t) -> Self {
         Self { ptr }
     }
 
     /// Assign a [`Tensor`] to the input on the model.
-    pub fn set_tensor(&mut self, name: &str, tensor: &Tensor) -> Result<()> {
+    pub fn set_tensor(&mut self, name: &str, tensor: Tensor) -> Result<()> {
```

Comment:
Can't remember why we need to own this tensor here?

Reply:
Done